createTestsFromFeature.js

```diff
@@ -1,17 +1,47 @@
 const { createTestFromScenario } = require("./createTestFromScenario");
+const { shouldProceedCurrentStep, getEnvTags } = require("./tagsHelper");
 
 const createTestsFromFeature = parsedFeature => {
+  const featureTags = parsedFeature.feature.tags;
+  const hasEnvTags = !!getEnvTags();
+  const hasFeatureTags = featureTags && featureTags.length > 0;
+
+  let featureShouldRun = true;
+  if (hasEnvTags) {
+    if (hasFeatureTags) {
+      featureShouldRun = shouldProceedCurrentStep(featureTags);
+    } else {
+      featureShouldRun = false;
+    }
+  }
+
+  const scenariosHaveTags = parsedFeature.feature.children.some(
```

Comment:
this seems more like scenarioShouldRun? we need the scenario to have specific tags, not any tags